### PR TITLE
Hide other layer when NGI layer is selected

### PR DESCRIPTION
**Pull request by danangmassandy**
This is for #2022.
Preview:
Peek 2024-03-06 10-01

Also add cache-control header to the NGI layer API.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 10 min

File: django_project/frontend/src/reducers/LayerFilter.ts [hunk 1]
```diff
@@ -28,10 +28,35 @@ export const LayerFilterSlice = createSlice({
             state.contextLayers = [..._layers]
         },
         toggleLayer: (state, action: PayloadAction<number>) => {
+            let _payloadLayers = state.contextLayers.filter(a => a.id === action.payload)
+            let _foundLayer: ContextLayerInterface = null;
+            if (_payloadLayers.length) {
+                _foundLayer = {..._payloadLayers[0]}
+            }
+            console.log('foundLayer : ', _foundLayer)
```
Comment (dimasciput) [inline django_project/frontend/src/reducers/LayerFilter.ts:36]:
```suggestion
```

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 11 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 10 min

**Pull request merged**